src/screens/App/screens/Dashboard/components/Filter/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import './style.css'
+
+const Filter = (props) => {
```

Comment:
use destructuring in functions
`const Filter = ({ filter }) => { `

Reply:
so you havent to do `props.param` all the time and is more descriptive

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment